style.css

```diff
@@ -98,10 +95,319 @@ h1 {
+  gap: 50px 0;
+  width: 317px;
+}
+
+.w-image {
+  display: block;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+h2 {
+  font-size: 32px;
+  font-family: 'Crete Round', serif;
+}
+
+.p-description {
+  font-size: 16px;
+  margin-bottom: 10px;
+}
+
+.languages {
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  margin-bottom: 10px;
+  width: 236px;
+  background: white;
+  align-items: center;
+}
+
+li .languages {
+  flex: 1;
+}
+
+.p-btn {
+  background: rgba(255, 107, 0, 1);
+  color: white;
+  font-size: 17px;
+  border: none;
+  height: 48px;
+  width: 124px;
+  font-weight: bold;
+}
+
+.item1 {
+  grid-area: card1;
+}
+
+.item2 {
+  grid-area: card2;
+}
+
+.item3 {
+  grid-area: card3;
+}
+
+.item4 {
+  grid-area: card4;
+}
+
+.item5 {
+  grid-area: card5;
+}
+
+.item6 {
+  grid-area: card6;
+}
+
+.card-works {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+  grid-template-areas:
+    "card1 card1 card1 card1 card1 card1"
+    "card2 card2 card2 card2 card2 card2"
+    "card3 card3 card3 card3 card3 card3"
+    "card4 card4 card4 card4 card4 card4"
+    "card5 card5 card5 card5 card5 card5"
+    "card6 card6 card6 card6 card6 card6";
+  grid-gap: 20px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: url(images/portfolio.png) no-repeat;
+  width: 317px;
+  height: 386px;
+  margin: 0 auto;
+  margin-bottom: 40px;
+}
+
+.text-block {
+  padding: 38px 16px;
+}
+
+.card-description {
+  color: white;
+  font-size: 16px;
+}
+
+.new {
+  color: white;
+  font-size: 32px;
+  margin-top: 40px;
+}
+
+.card-btn {
+  display: block;
+  background: rgba(255, 107, 0, 1);
+  min-height: 48px;
+  color: white;
+  font-size: 17px;
+  border: none;
+  font-weight: bold;
+}
+
+.lang {
+  display: flex;
+  flex-direction: row;
+  margin-right: 10px;
+  width: 236px;
   list-style: none;
+
+  /* background: rgba(255, 255, 255, 0.24); */
+  align-items: center;
+  color: white;
+  margin-top: 10px;
+}
+
+li .lang {
+  flex: 1;
+  text-align: center;
+  padding: 1px;
+}
+
+.title {
+  width: 315px;
+  left: 24px;
+  top: 120px;
+  align-items: center;
+}
+
+span {
+  font-size: 40px;
+  color: rgba(28, 26, 25, 1);
+  line-height: 52px;
+}
+
+.about-section {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  background: url(images/Rectangle\ 73.png);
+  width: 100%;
+  height: 1901px;
+}
+
+.about-descript {
+  color: white;
+  font-size: 16px;
+  margin-top: 24px;
+}
+
+.abt-btn {
+  background: rgba(255, 107, 0, 1);
+  order: 2;
+  width: 161px;
+  height: 48px;
   margin-top: 20px;
-  margin-right: 200px;
+  margin-bottom: 100px;
+  color: white;
+  font-size: 17px;
+  border: none;
+  font-weight: bold;
+}
+
+.about {
+  width: 100%;
+  padding: 120px 24px;
+}
+
+.spaced {
+  font-size: 72px;
+  width: 327px;
+  height: 79px;
+}
+
+.about-2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

Comment:
CSS styles that are common to more than one element should be grouped together into a class so that they can be added to whatever element needs them. Repeating them several time does not conform to the DRY principle.

Reply:
I don't understand this...I would be grateful if you can give me a clue